java/ingest/ingest-runner/src/main/java/sleeper/ingest/impl/IngestCoordinator.java

```diff
@@ -491,6 +497,18 @@ public Builder<T> ingestFileWritingStrategy(IngestFileWritingStrategy ingestFile
             return this;
         }
 
+        /**
+         * Determines whether the results of the ingest job will be committed to the state store by the ingest
+         * coordinator (synchonously), or by sending them to the state store committer lambda (asynchrnously).
```

Comment:
Spelling: "synchonously" should be "synchronously"; "asynchrnously" should be "asynchronously".